src/Uno.UI.Composition/Composition/Compositor.cs

```diff
@@ -45,6 +45,11 @@ public CompositionScopedBatch CreateScopedBatch(CompositionBatchTypes batchType)
 		public ShapeVisual CreateShapeVisual()
 			=> new ShapeVisual(this);
 
+#if __SKIA__
+		internal BorderVisual CreateBorderShapeVisual()
+			=> new BorderVisual(this);
+#endif
```

Comment:
Should this be in Compositor.skia.cs?

Reply:
It doesn't really matter, but I prefer to have it next to the other `CreateXX` methods